Classes/Hooks/DataHandler.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+/**
+ * @license GPLv3, http://www.gnu.org/copyleft/gpl.html
+ * @copyright Gilbertsoft (gilbertsoft.org), 2017
+ * @copyright Aimeos (aimeos.org), 2017-
+ * @package TYPO3
+ */
+
+namespace Aimeos\Aimeos\Hooks;

```

Comment:
Hm, the class for RealUrl auto-configuration is also in Classes/Custom but used the `$GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']` array as well as the scheduler configuration but the both are no hooks in the strict meaning. The intention for the "Custom" directory/namespace was to have all classes in place that doesn't inherit from another class or implement an interface and where the namespace isn't strictly defined. Therefore, it would be consequent to store the datahandler class there as well.